### ISSUE TITLE
b2: on cleanup delete hide marker it is the current file

### ISSUE THREAD

**Issue by ncw** [ncw/rclone#604]
If we don't do that then we get left with the hide markers which show in the web interface.

```
$ mkdir /tmp/cleanup-test
$ echo one > /tmp/cleanup-test/file1.txt
$ rclone -q sync /tmp/cleanup-test b2:cleanup-test
$ rclone --b2-versions -q ls b2:cleanup-test
4 file1.txt
$ rm /tmp/cleanup-test/file1.txt
$ rclone -q sync /tmp/cleanup-test b2:cleanup-test
$ rclone --b2-versions -q ls b2:cleanup-test
4 file1-v2016-08-02-133808-000.txt
$ rclone -v cleanup b2:cleanup-test
2016/08/02 14:39:08 file1.txt: Not deleting current version (id "4_z9f821d93605dab7b5f580e1b_f106f5a6bb6795c41_d20160802_m133842_c001_v0001031_t0029") "hide"
2016/08/02 14:39:08 file1.txt: Deleting (id "4_z9f821d93605dab7b5f580e1b_f106823c0648d1986_d20160802_m133808_c001_v0001008_t0000")
$ rclone --b2-versions -q ls b2:cleanup-test
$
```

However if I look in the b2 interface I can see that it says "file1.txt (hidden)"

So on cleanup we need to delete the current file if it is in status `hide` too.

**Comment by GrahamCobb** [comment]
I just came here to report this bug, only to find you are already ahead of me!  Looking forward to the fix in v1.33. Thanks

**Comment by ncw** [comment]
This is fixed now it will be in the v1.33 release

Here is a beta with the fix: http://pub.rclone.org/v1.32-22-gb4f2ada%CE%B2/
